backend/file_io.py

```diff
@@ -39,7 +39,7 @@ def log(*msg):
         tstamp = datetime.datetime.now()
         try:
             logfile = None
-            logfile = open('logfile.txt',"a")
+            logfile = open('otone_data/logfile.txt',"a")
```

Comment:
What does this line do when running as an executable on windows/mac? where's the file created..